sovtokenfees/sovtokenfees/fees_authorizer.py

```diff
@@ -98,7 +99,7 @@ def _validate_fees_can_pay(self, request, inputs, outputs, required_fees):
         else:
             change_amount = sum([a[AMOUNT] for a in outputs])
             expected_amount = change_amount + required_fees
-            TokenReqHandler.validate_given_inputs_outputs(
+            validate_given_inputs_outputs(
```

Comment:
I think, it would be better to stay this functionality as a static method in TokenReqHandler.

Reply:
TokenReqHandler will be removed, did you ment TokenBatchHandler?